olp-cpp-sdk-core/include/olp/core/cache/CacheSettings.h

```diff
@@ -31,14 +31,17 @@ namespace cache {
  * @brief Options for opening the disk cache
  */
 enum OpenOptions : unsigned char {
-  Default = 0x00,
-  ReadOnly = 0x01,
-  CheckCrc = 0x02
+  Default = 0x00, /*!< Opens the disk cache in read-write mode */
+  ReadOnly = 0x01, /*!< Opens the disk cache in read only mode */
+  CheckCrc = 0x02 /*!< Verifies checksum of all data that is read */
 };
 
+/**
+ * @brief Settings for in-memory and on-disk caching.
+ */
 struct CacheSettings {
   /**
-   * @brief Path to save contents on disk
+   * @brief Path to the on-disk cache.
```

Comment:
Please also specify that if you leave this path empty we will only create an in-memory cache that is limited by `max_memory_cache_size`.